Formula/openclonk.rb

```diff
@@ -22,26 +47,76 @@ class Openclonk < Formula
     sha256 cellar: :any, high_sierra:    "87779de2d3cfa0dc1880fa45226e3f434ecca4409565db5e8bf278c225487da1"
   end
 
+  head do
+    url "https://github.com/openclonk/openclonk.git", branch: "master"
+
+    depends_on "libepoxy"
+
+    uses_from_macos "curl"
+
+    on_linux do
+      depends_on "miniupnpc"
+      depends_on "sdl2"
+    end
+  end
+
   depends_on "cmake" => :build
-  depends_on "boost"
   depends_on "freealut"
   depends_on "freetype"
-  depends_on "glew"
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "libogg"
   depends_on "libpng"
   depends_on "libvorbis"
+  depends_on "tinyxml"
+
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "gcc"
```

Comment:
Based on GCC plan, we probably don't want to add this and we will be blocked on Linux bottling until we can get `mesa` rebottled (dropping in #106975) with a system GCC (either GCC-5 by disabling some drivers/features or by upgrading system Ubuntu/GCC).

Mesa still claims to build with GCC 4.2.0 but may require some parts to be disabled: https://docs.mesa3d.org/install.html#compiler